modules/concom/vue/views/StaffStructurePage.js

```diff
@@ -1,13 +1,15 @@
 /* globals Vue, showSpinner, hideSpinner */
 import { useCurrentUser } from '../composables/currentUser.js';
 import { useDivisionHierarchy } from '../composables/divisionHierarchy.js';
+import { useSidebar } from '../composables/sidebar.js';
+import { useSidebarActions } from '../composables/sidebarActions.js';
 
 const TEMPLATE = `
   <div class="UI-container">
     <div :class="sidebarDisplayClass">
       <div class="CONCOM-admin-content">
         <div class="CONCOM-admin-section">
-          <span>ConCom Structure</span>
+          <span>ConCom Structure (NEW)</span>
```

Comment:
I will fix this in a future PR. I am currently using this while testing and committed it by mistake.